src/frontend/App/context/BehandlingContext.tsx

```diff
@@ -62,6 +62,7 @@ const [BehandlingProvider, useBehandling] = constate(() => {
     }, []);
 
     const [visHenleggModal, settVisHenleggModal] = useState(false);
+    const [visSettpåVentModal, settVisSettpåVentModal] = useState(false);
```

Comment:
Stort P for På? 
```suggestion
    const [visSettPåVentModal, settVisSettPåVentModal] = useState(false);
```

Reply:
Vi så ikke denne før den ble merget inn, men det kommer en ny PR for resten av oppgaven. Legger det inn der!😁